schema/properties_directory/non-scalar/hubbard_v.json

```diff
@@ -24,6 +24,7 @@
                     "id2",
                     "atomicSpecies",
                     "atomicSpecies2",
+                    "distance",
```

Comment:
Let's have v and v_nn to be identical from the data structure standpoint, the latter is just for convenience purposes.

We could do one of the following:

- get the default highest occupied orbital for each element per https://github.com/Exabyte-io/periodic-table.js/blob/dev/periodic-table.json and insert during parsing
- try to read this information from XML file when dealing with QE